pkg/data/manifests/manifests_asset.go

```diff
@@ -0,0 +1,53 @@
+package manifests
+
+import (
+	"bytes"
+	"embed"
+	"fmt"
+
+	"github.com/openshift/installer/pkg/asset/ignition/bootstrap"
+	"github.com/openshift/installer/pkg/types"
+	"github.com/vincent-petithory/dataurl"
+
+	bootstrapfiles "github.com/openshift/installer-aro-wrapper/pkg/data/bootstrap"
+)
+
+//go:embed opt/*
+var assets embed.FS
+
+type ManifestsConfig struct {
+	AROWorkerRegistries string
+	HTTPSecret          string
+	AccountName         string
+	ContainerName       string
+	CloudName           string
+	AROIngressInternal  bool
+	AROIngressIP        string
+}
+
+func AppendManifestsFilesToBootstrap(bootstrap *bootstrap.Bootstrap, manifestsConfig ManifestsConfig) error {
+	err := bootstrapfiles.AddStorageFiles(bootstrap.Config, "opt", "opt", manifestsConfig, assets)
```

Comment:
Reuses the function used in mdsd.
https://github.com/openshift/installer-aro-wrapper/blob/d6af50321aced4a322a645539afa39420b679469/pkg/data/bootstrap/util.go#L17